drop-boxes/test-dropbox/etl_msconvert.py

```diff
@@ -484,4 +484,4 @@ def process(transaction):
     transaction.moveFile(incomingPath, dataSet)
 
     # Update Sample Status
-    sampleTracker.updateSampleLocationToCurrentLocation(code)
+    sampleTracker.updateSampleStatus(code)
```

Comment:
here you forgot to update the variables like in the files above?

Reply:
not sure what this script did here, was probably a test, removed